great_expectations/rule_based_profiler/parameter_builder/numeric_metric_range_multi_batch_parameter_builder.py

```diff
@@ -100,6 +100,10 @@ class NumericMetricRangeMultiBatchParameterBuilder(MetricMultiBatchParameterBuil
         "upper_bound",
     }
 
+    METRIC_NAMES_EXEMPT_FROM_VALUE_ROUNDING: set = {
```

Comment:
This looks like it becomes a whitelist of metrics that we don't want to round? Should this be a class attribute of the metric itself? That way if a metric is developed someone can set that directly on the metric inside of knowing they have to add it to this whitelist. It may be easier for contributors.

Reply:
@billdirks While this sounds cool, I do not feel that a metric should be concerned with the various operations that can be performed on it.  It is really the concern of the given range estimator (nothing less right now).  In other words, some other operation may not even care about the notion of rounding, or "do it anyway" for its own purposes.  Happy to discuss.  Thank you.

Reply:
Ok. If we encounter this often we may want to generalize this somehow. I don't know this part of the  codebase well enough to suggest a better solution.